packages/front-end/components/Features/FeatureModal/index.tsx

```diff
@@ -158,33 +148,30 @@ export default function FeatureModal({
   const { hasCommercialFeature } = useUser();
   const { requireProjectForFeatures } = useOrgSettings();
 
-  const customFields = filterCustomFieldsForSectionAndProject(
-    useCustomFields(),
-    "feature",
-    project,
-  );
-
-  const holdoutsEnabled = useFeatureIsOn("holdouts_feature");
-
   const defaultValues = genFormDefaultValues({
     environments,
     permissions: permissionsUtil,
     featureToDuplicate,
     project,
-    customFields: hasCommercialFeature("custom-metadata")
-      ? customFields
-      : undefined,
   });
 
   const form = useForm({ defaultValues });
 
+  const selectedProject = form.watch("project");
+  const customFields = filterCustomFieldsForSectionAndProject(
+    useCustomFields(),
+    "feature",
+    selectedProject,
```

Comment:
I understand changing `project` to `selectedProject`.  I'm unclear about all the other changes in this PR.

These are the main questions I have when looking at all of the other changes:
- Does removing the `hasCommercialFeature` check mean that custom fields are now available to all plans?
- If the feature being duplicated is missing some of the custom fields, what happens? (e.g. you add a new custom field after a feature was created and never update that feature, then you duplicate it)
- If you change projects, it doesn't look like any field values are updated. Does this actually work or does it only fix itself if you then go and edit the custom field values?

I haven't tested anything myself, so it could all be fine. Just, without context, these changes seem a little risky.